app/src/main/res/layout/pdp_fragment.xml

```diff
@@ -1,150 +1,152 @@
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        android:src="@drawable/image_product" />
+    <data>
 
-    <View
-        android:id="@+id/view"
-        android:layout_width="0dp"
-        android:layout_height="0dp"
-        app:layout_constraintTop_toBottomOf="@id/iv_product"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        android:layout_marginTop="@dimen/normal_150"
-        android:background="@drawable/ic_rectangle_1" />
+        <variable
+            name="viewModel"
+            type="com.example.trempel.PdpViewModel" />
+    </data>
 
-    <TextView
-        android:id="@+id/tv_product_title"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/normal_150"
-        android:layout_marginTop="@dimen/normal_150"
-        android:layout_marginEnd="@dimen/normal_150"
-        android:lineHeight="@dimen/normal_100"
-        android:fontFamily="@font/montserrat_black"
-        android:textColor="@color/white"
-        android:textSize="@dimen/text_size_product_title"
-        app:layout_constraintEnd_toStartOf="@id/tv_product_price"
-        app:layout_constraintStart_toStartOf="@id/view"
-        app:layout_constraintTop_toTopOf="@id/view"
-        android:text="@string/product_title"
-        />
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/white">
 
-    <TextView
-        android:id="@+id/tv_product_price"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="@dimen/normal_150"
-        android:layout_marginEnd="@dimen/normal_150"
-        android:font="@font/montserrat_light"
-        android:lineHeight="@dimen/normal_150"
-        android:textColor="@color/white"
-        android:textSize="@dimen/text_size_product_price"
-        app:layout_constraintEnd_toEndOf="@id/view"
-        app:layout_constraintTop_toTopOf="@id/view"
-        android:text="@string/product_price"
-        />
+        <ImageView
+            android:id="@+id/iv_product"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/height_iV_price"
+            android:layout_marginStart="@dimen/large_250"
+            android:layout_marginTop="@dimen/normal_150"
+            android:layout_marginEnd="@dimen/large_250"
+            android:scaleType="centerInside"
+            android:src="@{viewModel.product.image}"
+            app:imageUrl="@{viewModel.product.image}"
```

Comment:
why do you need both `src` and `imageUrl`?

Reply:
done